console.log('program starts ')
try{
var a=10
var c=a+b 
console.log(`sum of ${a} and ${b} is ${c} `);
}
catch(e){
    console.log('error is '+e);
}
console.log('program ends ')